zhaquirks/tuya/ts000x.py

```diff
@@ -322,6 +322,98 @@ class Switch_2G_Metering(EnchantedDevice, CustomDevice):
     }
 
 
+class Switch_2G_Var03(EnchantedDevice, CustomDevice):
```

Comment:
I think just `EnchantedDevice` is enough, as that also extends `CustomDevice`. But I see that it's also done like this in other places, so I guess it's fine too

Reply:
Yes, you're right. And have warned me already 😞 
I'll update all of them (in this file) in another PR

Reply:
I think some one was doing most or all of them 😞